openconnect_sso/config.py

```diff
@@ -112,6 +123,22 @@ def password(self, value):
         except keyring.errors.KeyringError:
             logger.info("Cannot save password to keyring.")
 
+    @property
+    def totp(self):
+        try:
+            totpsecret = keyring.get_password(APP_NAME, "totp/" + self.username)
+            return pyotp.TOTP(totpsecret).now()
```

Comment:
Needs check if `totpsecret` is defined, e.g.:
`return pyotp.TOTP(totpsecret).now() if totpsecret else None`